src/main/resources/assets/design-system/margin-padding.scss

```diff
@@ -1,15 +1,27 @@
 :root {
   --sci-margin-1: calc(0.5 * var(--sci-base-unit)); // Default: 8px / 0.5rem
-  --sci-margin-05: calc(0.5 * var(--sci-margin-1)); // Default: 4px / 0.25rem
+  --sci-margin-0_5: calc(0.5 * var(--sci-margin-1)); // Default: 4px / 0.25rem
+  --sci-margin-1_5: calc(1.5 * var(--sci-margin-1)); // Default: 12px / 0.75rem
   --sci-margin-2: calc(2 * var(--sci-margin-1)); // Default: 16px / 1rem
   --sci-margin-3: calc(3 * var(--sci-margin-1)); // Default: 24px / 1.5rem
   --sci-margin-4: calc(4 * var(--sci-margin-1)); // Default: 32px / 2rem
+
+  // Alternative variable for backwards compatability
+  --sci-margin-05: --sci-margin-0_5; // Default: 4px / 0.25rem
```

Comment:
Can also live with removing this var, marking it as breaking and renaming the few places where we use it in OX

Reply:
Alighty